addons/text_identification/src/presenter.js

```diff
@@ -536,9 +537,11 @@ function Addontext_identification_create() {
         } else if (eventName == "HideAnswers") {
             presenter.hideAnswers();
         } else if (eventName === "GradualShowAnswers") {
+            presenter.GSAcounter++;
             if (!presenter.isGradualShowAnswersActive) {
                 presenter.isGradualShowAnswersActive = true;
-            } 
+            }
+            if(presenter.GSAcounter === 1) presenter.hideStudentAnswersForGSA();
```

Comment:
Event GradualShowAnswer jest odbierany przez wszystkie addony TextIdentification i w jednym momencie wszystkie wykonują metodę hideStudentAnswersForGSA() 

Zachowanie addonu miało działać na zasadzie gdy po raz pierwszy GSA "dojdzie" do tego addonu, to w tym momencie odpowiedzi użytkownika są chowane. Czyli dopiero po ifie (data.moduleID === presenter.configuration.addonID

Reply:
Jeśli masz na myśli że dosłownie po ifie ma się to wykonywać, to nie widzę sensu przemieszczania jednej linijki kodu trochę dalej, tym bardziej, że i tak działa. Jeśli natomiast chodzi Ci o to, żeby się to wykonywało jako część tego ifa, to wtedy nigdy się nie schowają niepoprawne odpowiedzi

Reply:
Dla tego przypadku GSA musi ukryć na raz wszystkie odpowiedzi użytkownika.